app/views/OverviewPageView.scala.html

```diff
@@ -0,0 +1,99 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.FrontendAppConfig
+@import views.html.headerFooterTemplate.Layout
+
+@this(layout: Layout)
+
+@(isAgent: Boolean)(implicit request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
+
+@breadcrumbs = {
+<div class="govuk-breadcrumbs ">
+ <ol class="govuk-breadcrumbs__list">
+  <li class="govuk-breadcrumbs__list-item">
+   <a class="govuk-breadcrumbs__link" href="#">Income Tax</a>
+  </li>
+  <li class="govuk-breadcrumbs__list-item">
+   <a class="govuk-breadcrumbs__link" href="#">Income Tax Submission</a>
+  </li>
+  <li class="govuk-breadcrumbs__list-item">
+   <a class="govuk-breadcrumbs__link">2020 to 2021 Income Tax</a>
+  </li>
+ </ol>
+</div>
+}
+
+@taskWithActionList(heading: String, action: String) = {
+<li>
+ <h2 class="app-task-list__section">
+  <span class="app-task-list__section-number">1. </span> @heading
+ </h2>
+ <ol class="app-task-list__items">
+  <li class="app-task-list__item">
+              <span class="app-task-list__task-name">
+                <a href="#">
+                  @action
+                </a>
+              </span>
+   <span class="hmrc-status-tag">Not Complete</span>
+  </li>
+ </ol>
+</li>
+}
+
+@taskWithCaption(heading: String, caption: String) = {
+ <li>
+  <h2 class="app-task-list__section">
+   <span class="app-task-list__section-number">2. </span> @heading
```

Comment:
There's a slight issue here with the numbering as '2' is repeated 
<img width="731" alt="Screenshot 2020-09-25 at 15 14 19" src="https://user-images.githubusercontent.com/60218795/94277683-cd448b80-ff41-11ea-8c2b-1d0007d30431.png">